bigquery_etl/glean_usage/templates/metrics_clients_daily.query.sql

```diff
@@ -6,6 +6,7 @@
     DATE(submission_timestamp) AS submission_date,
     client_info.client_id AS client_id,
     sample_id,
+    normalized_channel,
```

Comment:
I expect `normalized_channel` here is often null. I think we want to inject `channel` here from the probeinfo data instead.